### ISSUE TITLE
unable to log in to admin or member ? 

### ISSUE THREAD

**Issue by Lordblanca** [WWBN/AVideo#6530]
When I log in to either admin or member I get a blank screen. 
![image](https://user-images.githubusercontent.com/37937968/164838022-dee1423e-f154-4520-99f2-a3f8a9f9a901.png)

And the side menu does not work.
![image](https://user-images.githubusercontent.com/37937968/164838114-43a85491-bf4f-4a87-9d3d-0504fb969ae6.png)

Unable to get to log file to see what is wrong

**Comment by DanielnetoDotCom** [comment]
I suspect you have a missing update.

get the logs thru the command line

`tail -n 200 -f /var/www/html/AVideo/videos/avideo.log`

**Comment by Lordblanca** [comment]
No that was not the problem. I had updated to chat 9.7 and I disabled the animation. An then I could not get to the site. I keep a manual log of everything I do. Before any changes I do a snap shot of the server. I restored from the snap shot and updated the chat and the disabled the animation again and all is well. I guess it was a bad update I've had that happen before with other softwares that I've installed on other machines. All good now. just figured I let you know. And thank you for the command line log command
